app/src/Sidebar/components/CategoricalBuy/BuySummary.js

```diff
@@ -45,14 +52,20 @@ const BuySummary = ({
     (stagedTradePositionGroups || []).forEach(
       ({ outcomeSet, runningAmount }) => {
         let hasEnteredInvestment;
+        let investMentFee;
 
         try {
           const decimalInvest = Decimal(investmentAmount);
           hasEnteredInvestment = decimalInvest.gt(0);
+          investMentFee = decimalInvest.mul(lmsrState.fee);
         } catch (err) {
           //
         }
 
+        humanReadablePositions.payOutWhen.fee = investmentAmount
+          ? investMentFee
```

Comment:
`investMent`?

Reply:
Added a typo in first instance and then the auto complete function worked for me :laughing: